app/src/main/java/com/android_academy/covid_19/ui/map/MapManager.kt

```diff
@@ -80,7 +80,9 @@ class MapManagerImpl(
     }
 
     override fun onCoronaChanged(markerOptions: List<LocationMarkerData>?) {
-        map.clear()
+        if (::map.isInitialized) {
+            map.clear()
+        }
 
         markerOptions?.forEach { options ->
```

Comment:
What about this one?
If map is not initialized, can you add markers?

Reply:
done...